src/app/loadout-builder/generated-sets/TierSelect.tsx

```diff
@@ -135,44 +140,61 @@ function MinMaxSelectInner({
   type,
   min,
   max,
+  ignored,
   stats,
   handleTierChange
 }: {
   stat: string;
   type: 'Min' | 'Max';
   min: number;
   max: number;
-  stats: { [statType in StatTypes]: MinMax };
+  ignored: boolean;
+  stats: { [statType in StatTypes]: MinMaxIgnored };
   handleTierChange(which: string, changed: any): void;
 }) {
   function handleChange(e) {
-    const value = parseInt(e.target.value, 10);
-    const lower = type.toLowerCase();
-    const opposite = lower === 'min' ? 'max' : 'min';
-    const update = {
-      [lower]: value,
-      [opposite]:
-        opposite === 'min' ? Math.min(stats[stat].min, value) : Math.max(stats[stat].max, value)
-    };
+    let update;
+    if (e.target.value === 'ignore' || e.target.value === 'include') {
+      update = {
+        min: stats[stat].min,
+        max: stats[stat].max,
+        ignored: !stats[stat].ignored
+      };
+    } else {
+      const value = parseInt(e.target.value, 10);
+      const lower = type.toLowerCase();
+      const opposite = lower === 'min' ? 'max' : 'min';
+      update = {
+        [lower]: value,
+        [opposite]:
+          opposite === 'min' ? Math.min(stats[stat].min, value) : Math.max(stats[stat].max, value),
+        ignored: false
+      };
+    }
+
     handleTierChange(stat, update);
   }
 
   const value = type === 'Min' ? Math.max(min, stats[stat].min) : Math.min(max, stats[stat].max);
-
   return (
-    <select value={value} onChange={handleChange}>
+    <select value={ignored ? '-' : value} onChange={handleChange}>
       <option disabled={true}>{t(`LoadoutBuilder.Select${type}`)}</option>
-      {/*
-        t('LoadoutBuilder.SelectMin')
-        t('LoadoutBuilder.SelectMax')
-       */}
```

Comment:
put this back

Reply:
Done, is it just to point out what the translations above can be?